CHANGES.rst

```diff
@@ -138,6 +138,12 @@ ramp_fitting
   of groups in a segment are computed when applying optimal weighting to line
   fit segments. [#7560, spacetelescope/stcal#163]
 
+- Updated the CI tests due to change in STCAL, which fixed bug for using the
```

Comment:
Because this didn't make it into the 1.11.0 release, this entry needs to be moved up to the top of the log.